src/index.ts

```diff
@@ -71,4 +71,5 @@ export * from 'semantic-ui-react'
 export * from './colors'
 
 // Types
+export { Network } from '@dcl/schemas'
```

Comment:
Question:
The other libs/dapps are importing these types (Network, ChainId, etc) directly from `@dcl/schemas` when needed. I don't see anything bad with this re-export except that it'll trigger VSCode's autocomplete 😅 .
You think that could be a problem?